Assets/MixedRealityToolkit/InputSystem/MixedRealityInputManager.cs

```diff
@@ -558,12 +558,12 @@ public void RaisePointerDown(IMixedRealityPointer pointer, Handedness handedness
         }
 
         /// <inheritdoc />
-        public void RaisePointerDown(IMixedRealityPointer pointer, Handedness handedness, InputType inputType)
+        public void RaisePointerDown(IMixedRealityPointer pointer, Handedness handedness, InputAction inputAction)
         {
             // Create input event
-            inputClickEventData.Initialize(pointer.InputSourceParent, handedness, inputType);
+            inputClickEventData.Initialize(pointer.InputSourceParent, handedness, inputAction);
 
-            if (inputType == InputType.Select)
+            if (inputAction.Description == "Select") // TODO - Unsure how the new InputActions are Referred to?
```

Comment:
Not sure string comparisons is the best way to do this.

Reply:
Lol, no, that was just one attempt, I used ID in another class, I'll fix that